setup.py

```diff
@@ -56,7 +56,7 @@
         "lru-dict>=1.1.6",
         # When updating to a newer version of pyrsistent, please check that the interface
```

Comment:
Since this doesnt seem like it will change ever again, should we remove this comment altogether?

Reply:
Removed!